package/e2e/__tests__/__snapshots__/check-parse-error.spec.ts.snap

```diff
@@ -11,14 +11,17 @@ exports[`check parse error "checkly test" should return a clear error when there
     The following NPM dependencies were used, but aren't supported in the 
     runtimes.
     For more information, see https://www.checklyhq.com/docs/runtimes/.
-    	entrypoint.js imports unsupported dependencies:
+    	/home/runner/work/checkly-cli/checkly-cli/package/e2e/__tests__/fixtures/c
```

Comment:
I think that it's nicer to have the filename in the error path be relative to the root of the project like it was before. Also since this is an absolute path, it's going to be failing outside of GH CI.

Reply:
Gotcha. That requires some work but I will take a look at it

Reply:
This is seriously annoying... Now, I broke other tests

Reply:
@clample the check-dependency-parser.spec.ts expects absolute path errors. We gotta make a decision here which way we want to support

Reply:
Got it 👍 I think that we can merge the PR with either absolute or relative path errors for now, whichever is easier.

We can discuss more with Tim and Stefan later on in #proj-local-workflow about if we should go with relative or absolute path errors in the long run.

Reply:
I'm gonna switch it back to absolute path given it is a smaller change